homeworks/maksim.sivachenko_MaxSiva/homework_5/innerCopy.js

```diff
@@ -0,0 +1,16 @@
+const obj = {
+    b: 'c',
+    d: { e: 'f' },
+};
+
+function innerCopy(myKey) {
+    const keys = Object.keys(myKey);
+    const result = {};
+    keys.forEach((key) => {
+        result[key] = myKey[key];
+    });
+
+    return result;
+}
```

Comment:
this will not do a deep copy... read about what is special about objects and arrays as complex type of data